src/drivers/rc/crsf_rc/CrsfRc.cpp

```diff
@@ -509,6 +557,17 @@ int CrsfRc::print_status()
 
 int CrsfRc::custom_command(int argc, char *argv[])
 {
+	const char *verb = argv[0];
+
+	if (!strcmp(verb, "bind")) {
+		uORB::Publication<vehicle_command_s> vehicle_command_pub{ORB_ID(vehicle_command)};
+		vehicle_command_s vcmd{};
+		vcmd.command = vehicle_command_s::VEHICLE_CMD_START_RX_PAIR;
```

Comment:
FYI this might get weird if other modules (px4io, other rc_input, etc) are running.